src/meander/substitute/epsilon.cljc

```diff
@@ -673,7 +676,7 @@
                     (if (r.subst.syntax/contains-cata-node? node)
                       `(try
                          [~form*]
-                         (catch Exception e#
+                         (catch (if-cljs js/Error ExceptionInfo) e#
```

Comment:
I think we could trim this down a bit by

```clj
#?(:clj (:import (clojure.lang Exception)))
```

in the `ns` above. And using `ExceptionInfo` here.